site/content/docs/main/file-system-backup.md

```diff
@@ -641,6 +641,16 @@ Both the uploader and repository consume remarkable CPU/memory during the backup
 Velero node-agent uses [BestEffort as the QoS][14] for node-agent pods (so no CPU/memory request/limit is set), so that backups/restores wouldn't fail due to resource throttling in any cases.  
 If you want to constraint the CPU/memory usage, you need to [customize the resource limits][15]. The CPU/memory consumption is always related to the scale of data to be backed up/restored, refer to [Performance Guidance][16] for more details, so it is highly recommended that you perform your own testing to find the best resource limits for your data.   
 
+For Kopia path, some memory is preserved by the node-agent to avoid frequent memory allocations, therefore, after you run a file-system backup/restore, you won't see node-agent releases all the memory. There is a limit for the memory preservation, so the memory won't increase all the time. The limit varies from the number of CPU cores in the cluster nodes, as calculated below:  
```

Comment:
Can we clarify how much if at all is released? Should there be timeout for preserved memory? If you only backup once every 6 months, you may rather spend time to reallocate memory next backup.

Reply:
Added the clarification that there is no timeout for the preserved memory, so the "memory is not released until node-agent restarts."

Reply:
> Can we clarify how much if at all is released?

The released memory is unknown actually, because `released memory = total allocated memory - preserved memory`. While `total allocated memory` is decided by the backup itself.

Reply:
> Should there be timeout for preserved memory? 

Yes, it is more rational to have a smarter mechanism instead of preserve the memory forever. But whether a timeout is ideal enough, we need to further consider.
At present, we just document it and leave it as is. We think it is not a high priority task, reasons:
1. It happens to fs-backup only from 1.15 on, because data movers will not run in the long-running node-agent pods.
2. The preserved memory won't reach to the limit very easily, normally it is less than the limit
3. The backup is usually a scheduled task, e.g., one/several per day, so the preserved memory is normally effective